### PR TITLE
D7-SDK-652: Update the plugin to use the Yoti PHP SDK version 2

### PR DESCRIPTION
**Pull request by sidibos**

### CODE REVIEW

File: yoti/yoti.module [hunk 1]
```diff
@@ -310,11 +311,6 @@ function yoti_user_view($account, $view_mode, $langcode) {
       }
     }
 
-    // Format date of birth.
```
Comment (echarrod) [inline yoti/yoti.module:313]: Are we just getting rid of this because it's being done in `processProfileAttributes`?

File: yoti/YotiHelper.php [hunk 2]
```diff
@@ -91,54 +93,49 @@ public function link($currentUser = NULL) {
...
-
-    if (!$this->passedAgeVerification($activityDetails)) {
+    if (!$this->passedAgeVerification($profile)) {
+      self::setFlash("Could not log you in as you haven't passed the age verification", 'error');
       return FALSE;
     }
 
     // Check if Yoti user exists.
-    $drupalYotiUid = $this->getDrupalUid($activityDetails->getUserId());
+    $drupalUid = $this->getDrupalUid($activityDetails->getRememberMeId());
```
Comment (echarrod) [inline yoti/YotiHelper.php:110]: How come we changed this from `drupalYotiUid` to `drupalUid`?

**Review by echarrod — APPROVED** [review]

File: yoti/YotiHelper.php (hunk 2 above)
Comment (sidibos) [reply yoti/YotiHelper.php:110]: Yes this the Drupal User Id, so it's clearer this way. It's just the naming.

File: yoti/yoti.module (hunk 1 above)
Comment (sidibos) [reply yoti/yoti.module:313]: Yes, it will be already formatted when it gets in here.

**Review by bucky-boy — APPROVED** [review]

**Pull request merged**